### PR TITLE
Remove Gogo-Proto From Testsuite Definitions

### PR DESCRIPTION
**Pull request by d80tb7**
Remove all of gogo proto from the test suite

### CODE REVIEW

File: internal/common/proto/protoutil.go [hunk 1]
```diff
@@ -85,8 +85,26 @@ func ToStdTime(ts *types.Timestamp) time.Time {
 }
 
 func ToTimestamp(t time.Time) *types.Timestamp {
+	types.TimestampNow()
```
Comment (robertdavidsmith) [inline internal/common/proto/protoutil.go:88]: ?

File: internal/common/proto/protoutil.go [hunk 2]
```diff
@@ -85,8 +85,26 @@ func ToStdTime(ts *types.Timestamp) time.Time {
...
 	return &types.Timestamp{
 		Seconds: t.Unix(),
 		Nanos:   int32(t.Nanosecond()),
 	}
 }
+
```
Comment (robertdavidsmith) [inline internal/common/proto/protoutil.go:94]: Unit test?

File: internal/executor/utilisation/cluster_utilisation.go [hunk 3]
```diff
@@ -3,6 +3,8 @@ package utilisation
 import (
 	"fmt"
 
+	armadaslices "github.com/armadaproject/armada/internal/common/slices"
```
Comment (robertdavidsmith) [inline internal/executor/utilisation/cluster_utilisation.go:6]: Move further down

File: pkg/executorapi/util.go [hunk 4]
```diff
@@ -3,6 +3,11 @@ package executorapi
 import (
 	"time"
 
+	v1 "k8s.io/api/core/v1"
```
Comment (robertdavidsmith) [inline pkg/executorapi/util.go:6]: Import ordering

File: pkg/executorapi/util.go [hunk 5]
```diff
@@ -64,3 +73,23 @@ func NewNodeFromNodeInfo(nodeInfo *NodeInfo, executor string, allowedPriorities
 		ReportingNodeType:                nodeInfo.NodeType,
 	}, nil
 }
+
+func ResourceListFromProtoResources(r map[string]*resource.Quantity) schedulerobjects.ResourceList {
```
Comment (robertdavidsmith) [inline pkg/executorapi/util.go:77]: unit test?

File: pkg/executorapi/util.go (hunk 4 above)
Comment (d80tb7) [reply pkg/executorapi/util.go:7]: I think  it's correct!

**Review by robertdavidsmith — APPROVED** [review]

**Pull request merged**